.env

```diff
@@ -0,0 +1,2 @@
+DATABASE_URL=postgresql://postgres:12345@localhost:5433/Epic-Mail
```

Comment:
your `.env` is still online. Please remove this file from github

Reply:
removed it from my branch